egs/wsj/s5/steps/nnet3/chain/build_tree.sh

```diff
@@ -158,17 +156,13 @@ if [ $stage -le -3 ] && $train_tree; then
      $lang/phones/sets.int $dir/questions.int || exit 1;
   cat $lang/phones/extra_questions.int >> $dir/questions.int
   $cmd $dir/log/compile_questions.log \
-    compile-questions --leftmost-questions-truncate=$leftmost_questions_truncate \
-      $context_opts $lang/topo $dir/questions.int $dir/questions.qst || exit 1;
+    compile-questions $context_opts $lang/topo \
+      $dir/questions.int $dir/questions.qst || exit 1;
 
   # questions_truncated.int will be needed later on when we build the phone
   # language model for 'chain' training.  It's a mechanism of keeping the graph
   # small.
```

Comment:
I guess `questions_truncated.int` is never used so this comment and the following `cp ...` line could be removed but not sure.